### PR TITLE
Create Web Link

### PR DESCRIPTION
**Pull request by Trudko**
Implementation, documentation and unit tests.

### CODE REVIEW

File: docs/web-links.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Web Links
+=========
+
+Web links are objects that point to URLs. These objects are also known as bookmarks within the Box web application.
+Web link objects are treated similarly to file objects, so they will also support shared links, copy, permanent delete,
+and restore with the format /web_links/{WEB_LINK_ID} in place of /files/{FILE_ID}
```
Comment (mattwiller) [inline docs/web-links.md]: It's probably unnecessary to reference the actual API endpoints in this SDK documentation.

Comment (Trudko) [inline docs/web-links.md]: I've removed the part mentioning endpoints.

File: docs/web-links.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Web Links
+=========
+
+Web links are objects that point to URLs. These objects are also known as bookmarks within the Box web application.
+Web link objects are treated similarly to file objects, so they will also support shared links, copy, permanent delete,
+and restore with the format.
```
Comment (mattwiller) [inline docs/web-links.md]: I think "with the format" doesn't belong here; what is it referring to?

**Comment by mattwiller** [comment]
Looks like there's a merge conflict from adding another manager that needs to be resolved before merging.

File: docs/web-links.md (hunk 2 above)
Comment (Trudko) [inline docs/web-links.md]: Yes you are correct with the format shouldn't be there. I've also resolved conflicts  with the master.

**Pull request merged**